test/integration/component/test_host_ha.py

```diff
@@ -17,22 +17,19 @@
 """ BVT tests for Hosts Maintenance
 """
 
-# Import Local Modules
-from marvin.codes import FAILED
 from marvin.cloudstackTestCase import *
-from marvin.cloudstackAPI import *
 from marvin.lib.utils import *
 from marvin.lib.base import *
 from marvin.lib.common import *
 from nose.plugins.attrib import attr
 
-from time import sleep
-
 _multiprocess_shared_ = False
 
 
 class TestHostHA(cloudstackTestCase):
 
+    hostCountMsg = "Host HA can be tested with at least two hosts, only %s found"
```

Comment:
not related but - isn't this VM HA? (not host HA with ipmi etc?)

Reply:
no? this is  test_host_ha, is it?

Reply:
Not I'm asking even though file says `test_host_ha.py`, are the tests for Host HA (or VM HA) that uses oobm/ipmi etc? Nevermind I'll check this myself.